pyproject.toml

```diff
@@ -20,6 +20,7 @@ lru-dict = "^1.1.7"
 toml = "^0.10.2"
 atomics = "^1.0.2"
 typing-extensions = "^4.5.0"
+types-pyyaml = "^6.0.0"
```

Comment:
Why is `types-pyyaml` not a dev-dependency? It is not needed outside of development.

I now notice that `typing-extensions` is also a regular dependency. We should make it a dev-dependency.

Reply:
moved